docs/PowerAuth-Push-Server-1.2.0.md

```diff
@@ -1,6 +1,34 @@
 # Migration from 1.1.0 to 1.2.0
 
-## Database changes
+## Database Changes
+
+### Per-App APNs Environment Config
+
+A new `ios_environment` column for the per-app APNs environment setting is introduced. The column can have the following values:
+
+- `NULL` value (or any unknown string value) - The global configuration is applied (keeps the current behavior).
+- `development` (case-insensitive) - The development environment is used for APNs. Use this value if the app is signed with development provisioning profile and distributed outside App Store and Testflight, i.e., via App Center.
+- `production` (case-insensitive) - The production environment is used for APNs. Use for App Store or Testflight builds.
+
+#### PostgreSQL
+
+```sql
+ALTER TABLE push_app_credentials ADD ios_environment VARCHAR(32) NULL;
+```
+
+#### Oracle
+
+```sql
+ALTER TABLE push_app_credentials ADD ios_environment VARCHAR2(32) NULL;
```

Comment:
I think it is better to use `VARCHAR2(32 CHAR)`. We should probably use it in all VARCHARs for Oracle in Push Server, because by default the limit is specified as number of bytes, not as number of characters.

Reply:
Good catch! I fixed this in b3632f8, including one `VARCHAR` vs. `VARCHAR2` issue.